app/scenes/Schedule/index.js

```diff
@@ -15,7 +15,7 @@ import moment from 'moment';
 
 import type { ScheduleTalk } from '../../types';
 
-import Splash from 'react-native-smart-splash-screen';
+// import Splash from 'react-native-smart-splash-screen';
```

Comment:
@blargity - `AnimatedListView` lets you run the animations in response to `onScroll` on the UI thread: http://facebook.github.io/react-native/blog/2017/02/14/using-native-driver-for-animated.html -- it makes the animation happen immediately in response to the scroll and so it doesn't drop frames when you page in new rows in the listview